sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationClientTestBase.java

```diff
@@ -86,8 +92,27 @@ String getLabel() {
         return testResourceNamer.randomName(labelPrefix, RESOURCE_LENGTH);
     }
 
+    /**
+     * Returns a stream of arguments that includes all combinations of eligible {@link HttpClient HttpClients} and
+     * service versions that should be tested.
+     *
+     * @return A stream of HttpClient and service version combinations to test.
+     */
+    protected Stream<Arguments> getTestParameters() {
```

Comment:
Can we have a permutation helper method in core-test? We can simple prepare the lists and pass into the method